src/mui/button/CreateButton.js

```diff
@@ -23,16 +23,20 @@ const styles = {
     },
 };
 
-const CreateButton = ({
+export const CreateButton = ({
     basePath = '',
     translate,
     label = 'aor.action.create',
     width,
+    disabled,
 }) =>
     width === 1 ? (
         <FloatingActionButton
             style={styles.floating}
-            containerElement={<Link to={`${basePath}/create`} />}
+            containerElement={
+                disabled ? 'div' : <Link to={`${basePath}/create`} />
```

Comment:
I think it doesn't make sense to show a disabled floating action button. Hide it in that case.

Reply:
Really? My goal is to display a disabled button, maybe we can add another property to hide button when disabled?

Reply:
Again, from a usability perspective, a "disabled" floating action button is an aberration IMO. A [floating action button](https://material.io/guidelines/components/buttons-floating-action-button.html) is used for a promoted action. As it floats over the content, if it's not active, it's just obscuring the content behind it.